### PR TITLE
Use the KuduScanToken API to get tablets

### PR DESCRIPTION
**Pull request by lzeiming**
KuduTable.getTabletsLocations() is deprecated and may cause the exception:

```
 java.lang.NullPointerException
 	at org.apache.kudu.client.AsyncKuduClient.discoverTablets(AsyncKuduClient.java:2335)
 	at org.apache.kudu.client.AsyncKuduClient$MasterLookupCB.call(AsyncKuduClient.java:2192)
 	at org.apache.kudu.client.AsyncKuduClient$MasterLookupCB.call(AsyncKuduClient.java:2173)
 	at com.stumbleupon.async.Deferred.doCall(Deferred.java:1280)
 	at com.stumbleupon.async.Deferred.addCallbacks(Deferred.java:685)
 	at com.stumbleupon.async.Deferred.addCallback(Deferred.java:721)
 	at org.apache.kudu.client.AsyncKuduClient.locateTablet(AsyncKuduClient.java:1726)
 	at org.apache.kudu.client.AsyncKuduClient.loopLocateTable(AsyncKuduClient.java:1851)
 	at org.apache.kudu.client.AsyncKuduClient.locateTable(AsyncKuduClient.java:1894)
 	at org.apache.kudu.client.AsyncKuduClient.syncLocateTable(AsyncKuduClient.java:1798)
 	at org.apache.kudu.client.KuduTable.getTabletsLocations(KuduTable.java:233)
 	at org.apache.kudu.client.KuduTable.getTabletsLocations(KuduTable.java:213)
 	at io.prestosql.plugin.kudu.properties.KuduTableProperties.getRangePartitionList(KuduTableProperties.java:338)
```

### CODE REVIEW

**Comment by kokosing** [comment]
What version of Kudu causes this issue?

**Comment by lzeiming** [comment]
We encounter this issue with Kudu 1.13.0 cluster and Kudu 1.10.0 Java client library. 
I think this is a bug in the Kudu Java client library.

File: plugin/trino-kudu/src/main/java/io/trino/plugin/kudu/properties/KuduTableProperties.java [hunk 1]
```diff
@@ -69,7 +66,7 @@
 
     private static final ObjectMapper mapper = new ObjectMapper();
 
-    private static final long DEFAULT_DEADLINE = 20000; // deadline for retrieving range partitions in milliseconds
+    private static final long DEFAULT_TIMEOUT = 20_000; // timeout for retrieving range partitions in milliseconds
```
Comment (Praveen2112) [inline plugin/trino-kudu/src/main/java/io/trino/plugin/kudu/properties/KuduTableProperties.java:69]: nit : Can we capture them as a separate commit ?

File: plugin/trino-kudu/src/main/java/io/trino/plugin/kudu/properties/KuduTableProperties.java [hunk 2]
```diff
@@ -365,7 +359,7 @@ private static RangeBoundValue buildRangePartitionBound(KuduTable table, byte[]
             Schema schema = table.getSchema();
             PartitionSchema partitionSchema = table.getPartitionSchema();
             PartitionSchema.RangeSchema rangeSchema = partitionSchema.getRangeSchema();
-            List<Integer> rangeColumns = rangeSchema.getColumns();
+            List<Integer> rangeColumns = rangeSchema.getColumnIds();
```
Comment (Praveen2112) [inline plugin/trino-kudu/src/main/java/io/trino/plugin/kudu/properties/KuduTableProperties.java:362]: This can also be a separate commit as `getColumns` is a deprecated API

**Review by Praveen2112 — APPROVED** [review]
Thanks for working on this. Can you please rebase your PR with the latest master.

File: plugin/trino-kudu/src/main/java/io/trino/plugin/kudu/properties/KuduTableProperties.java (hunk 1 above)
Comment (lzeiming) [reply plugin/trino-kudu/src/main/java/io/trino/plugin/kudu/properties/KuduTableProperties.java:69]: This change is because the method KuduTable.getRangePartitions(long timeout) use argument name timeout.

File: plugin/trino-kudu/src/main/java/io/trino/plugin/kudu/properties/KuduTableProperties.java (hunk 2 above)
Comment (lzeiming) [reply plugin/trino-kudu/src/main/java/io/trino/plugin/kudu/properties/KuduTableProperties.java:362]: Updated

Comment (Praveen2112) [reply plugin/trino-kudu/src/main/java/io/trino/plugin/kudu/properties/KuduTableProperties.java:362]: Can we add it as first commit ? Can we rename the commit message as `Remove deprecated API usage in KuduTableProperties`

File: plugin/trino-kudu/src/main/java/io/trino/plugin/kudu/properties/KuduTableProperties.java (hunk 1 above)
Comment (Praveen2112) [reply plugin/trino-kudu/src/main/java/io/trino/plugin/kudu/properties/KuduTableProperties.java:69]: Thanks. How about renaming the commit message - `Use KuduTable#getRangePartitions for partition details`

**Review by Praveen2112 — APPROVED** [review]

**Comment by Praveen2112** [comment]
Merged. Thanks for working on this
